server/interpreter/data/owner.go

```diff
@@ -18,12 +18,12 @@ package data
 
 // An Owner is an object that can own other objects and properties.
```

Comment:
Owner is a non-standard extension, so I think it warrants a slightly more detailed explanation which I have now provided.